tools/clickbench/createavg.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+def calculate_averages(input_file, output_file):
```

Comment:
nit: maybe add a comment describing what the input_file looks like so it's clear what this function's supposed to take the average of